x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils/import_rules_utils.ts

```diff
@@ -115,6 +115,7 @@ export const importRules = async ({
                 filters: filtersRest,
                 rule_id: ruleId,
                 index,
+                data_view_id: dataViewId,
```

Comment:
Similar to here, `data_view_id` needs to be wired through in `patch_rules_route.ts`, `patch_rules_bulk_route.ts`, `install_prepacked_rules.ts`, and `update_prepacked_rules.ts`